### DISCUSSION TITLE
New version

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#2011] [General]
Hello

I wanted to update to the new version and when making the changes in the _confy.yml file it shows the following:
![image](https://github.com/open-sdg/open-sdg/assets/117297044/4776826c-2e99-4cad-b6a2-b54e504bbc84)
![image](https://github.com/open-sdg/open-sdg/assets/117297044/d40e41b8-22bc-4242-b4d6-e1aa30877548)

And in the Gemfile the following:
![image](https://github.com/open-sdg/open-sdg/assets/117297044/eb0cf857-9462-49ad-8ab3-9274afe554d8)
![image](https://github.com/open-sdg/open-sdg/assets/117297044/f16314e6-8600-4984-8a85-4a294e93574a)

What are these errors?

Thank you

**Comment by brockfanning** [comment]
@SandraEliz The errors refer to not being able to reach the data - but it looks like you have upgraded to Ruby 3, which is not officially supported yet. (It will be supported soon, but not yet.) I could confirm if you could send a link to the code change that triggered this failure.

**Reply by brockfanning** [reply]
@SandraEliz Disregard my comment about Ruby 3 and the code change. I located your repository and I see that the problem is indeed that it cannot find your data repository. Did you perhaps make your data repository private?

**Reply by SandraEliz** [reply]
Hello

An apology I had not noticed that detail.

I had the data repository privately

Thank you
